test/functional/terminal/tui_spec.lua

```diff
@@ -438,34 +438,29 @@ describe('TUI FocusGained/FocusLost', function()
     feed_data(':set shell='..nvim_dir..'/shell-test\n')
     feed_data(':set noshowmode laststatus=0\n')
 
-    retry(2, 3 * screen.timeout, function()
-      feed_data(':terminal\n')
-      screen:sleep(1)
-      feed_data('\027[I')
-      screen:expect([[
-        {1:r}eady $                                           |
-        [Process exited 0]                                |
-                                                          |
-                                                          |
-                                                          |
-        gained                                            |
-        {3:-- TERMINAL --}                                    |
-      ]])
-      feed_data('\027[O')
-      screen:expect([[
-        {1:r}eady $                                           |
-        [Process exited 0]                                |
-                                                          |
-                                                          |
-                                                          |
-        lost                                              |
-        {3:-- TERMINAL --}                                    |
-      ]])
-
-      -- If retry is needed...
-      feed_data("\034\016")  -- CTRL-\ CTRL-N
-      feed_data(':bwipeout!\n')
-    end)
+    feed_data(':terminal\n')
+    screen:sleep(1)
+    feed_data('\027[I')
+    screen:expect({grid=[[
```

Comment:
for the "keyword args" form, convention is to omit the parens.

```suggestion
    screen:expect{grid=[[
```